testify/test_case.py

```diff
@@ -66,12 +68,8 @@ def _cmp_str(instance):
         """Return a canonical representation of a TestCase for sorting and hashing."""
         return "%s.%s" % (instance.__module__, instance.__name__)
 
-    def __cmp__(cls, other):
```

Comment:
I was actually the one who made this obsolete with this change here: https://github.com/Yelp/Testify/pull/216/files#diff-e4d2bf0d20a246b72b742afc1d4c3560R102